### PR TITLE
Pull request for virus Arbovirus chinensis

### PR DESCRIPTION
**Pull request by jjzjsjasdfa**
Completed template markdown file for Arbovirus_chinensis
Created img directory in virus directory
Created Arbovirus_chinensis directory in img directory to store the images used by the markdown file

### CODE REVIEW

File: tools/ORFfinder.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+**Tutorial Objective**: By finishing this tutorial, you will be able to use ORFfinder to find the open reading frame efficiently and meaningfully.
+
+
+## Input / Prerequisites
+- ORFfinder can be accessed from [here](https://www.ncbi.nlm.nih.gov/orffinder/).
+- The input could be accession numbers, data in fasta format, or bare sequences. See [Accepted Input Formats](https://blast.ncbi.nlm.nih.gov/doc/blast-topics/) for more details.
+- The web version of the ORF finder is limited up to 50 kb long. The stand-alone version, which doesn't have query sequence length limitation, is available for Linux x64.
+
+## Output
+1. Data of ORFs in different file formats.
```
Comment (ababaian) [inline tools/ORFfinder.md]: What kind of file formats?

File: tools/ORFfinder.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+output:
+  html_document: default
+  pdf_document: default
+---
+# ORFfinder
+written by: Nuan Wen
+
+[8 minutes] Welcome to the tutorial for ORFfinder. This tool helps researchers to find the open reading frame (ORF) quickly. An ORF is a sequence of DNA or RNA that potentially codes for a protein. This tool will find the ORF based on the start codon you specify until a stop codon appears. Additionally, it provides you with the corresponding amino acid sequences and shortcuts to BLAST, which makes the preparation work for virus discovery easier to carry out.
```
Comment (ababaian) [inline tools/ORFfinder.md]: What defines the "START" of an ORF? and what defines the "END" of an ORF?

File: virus/Arbovirus_chinensis.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Meganola albula, or Kent Black Arches, spans the Palearctic region, extending into eastern China, specifically Jiangxi and Sichuan provinces, situated in the middle of the Yangtze River [Boyes et al., 1775; Baidu, n.d.].
+
+The overlapping habitat near the Yangtze River suggests a potential avenue for virus transmission among these species, reflected in the shared presence of the RdRp sequence. The virus discovered may be an insect virus capable of infecting L. chinensis, Luehdorfia taibai, C. kiangsu, and M. albula, all of which are within the ecological range of M. musculus. This ecological connection could explain the presence of the virus in samples from these diverse species. However, the varying activity times—some species being diurnal and others nocturnal—may limit the extent of potential virus transmission due to differences in their active periods.
+```
+
+### Virus Genome {Q3}
```
Comment (ababaian) [inline virus/Arbovirus_chinensis.md]: Please use a more informative name for section titles.

Comment (ababaian) [reply virus/Arbovirus_chinensis.md]: Think of this more as a "report" of your findings, not as a "homework assignment"

File: virus/Arbovirus_chinensis.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+![ORF](img/Arbovirus/ORF.png)
+![protein domain](img/Arbovirus/protein_domain.png)
+![phylogenetic tree](img/Arbovirus/phylogenetic_tree_ClustalOmega.png)
+
+
+
+
+### Other (bonus) sections
```
Comment (ababaian) [inline virus/Arbovirus_chinensis.md]: "Bonus" is related to the homework, not to the reporting of findings.

File: virus/Arbovirus_chinensis.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Meganola albula, or Kent Black Arches, spans the Palearctic region, extending into eastern China, specifically Jiangxi and Sichuan provinces, situated in the middle of the Yangtze River [Boyes et al., 1775; Baidu, n.d.].
+
+The overlapping habitat near the Yangtze River suggests a potential avenue for virus transmission among these species, reflected in the shared presence of the RdRp sequence. The virus discovered may be an insect virus capable of infecting L. chinensis, Luehdorfia taibai, C. kiangsu, and M. albula, all of which are within the ecological range of M. musculus. This ecological connection could explain the presence of the virus in samples from these diverse species. However, the varying activity times—some species being diurnal and others nocturnal—may limit the extent of potential virus transmission due to differences in their active periods.
+```
+
+### Virus Genome {Q3}
+
+![ORF](img/Arbovirus/ORF.png)
```
Comment (ababaian) [inline virus/Arbovirus_chinensis.md]: Include a detailed Figure Legend for each Figure. Number Figures used throughout the report in the legend titles.

File: virus/Arbovirus_chinensis.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
+# An ugly bug in beautiful bugs, one novel Orthomyxoviridae virus identified in China
+written by: Nuan Wen
+
+### Arbovirus chinensis
+*The genus name "Arbovirus" indicates an arthropod-borne virus since I propose it is a virus transmitted between arthropods. The species name "chinensis," suggesting an association with China since it is mostly found in species endemic to China.*
+
+[Optional: Tutorial 2 Generative AI Cover-art](img/Arbovirus/cover.png)
```
Comment (ababaian) [inline virus/Arbovirus_chinensis.md]: Make this an embedded image, not a link.
 format is 
```
! [ Figure title ] ( link )

File: virus/img/.DS_Store
Comment (ababaian) [inline virus/img/.DS_Store:1]: Do not inlclude DOT files in the github repo

File: virus/img/Arbovirus_chinensis/IQTree.png
Comment (ababaian) [inline virus/img/Arbovirus_chinensis/IQTree.png:1]: In all the markdown links you used, you're using a different directory for images:
```
![phylogenetic tree](img/Arbovirus/phylogenetic_tree_ClustalOmega.png)
```

Please make everything consistent as use the path naming above `Arbovirus_chinesis/<image>` as the correct paths.

**Comment by jjzjsjasdfa** [comment]
Hi professor, I have revised based on your comments. Just want to note that my MSR uses all species in my virus family which only contains fewer than 10 species, so I was not randomly using a subset of species.

**Review by ababaian — APPROVED** [review]
Nice!

**Pull request merged**